test/integration/minishift.go

```diff
@@ -28,6 +28,11 @@ import (
 
 var lastCommandOutput CommandOutput
 
+const (
+	ConsoleURL   = "console --url"
```

Comment:
Its not about reusing. IMO its a good practice to create a constant variable rather expanding the value inside the function. As the features are growing there might be a chance to reuse it in a different function.